new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Helm.java

```diff
@@ -5,86 +5,175 @@
+        namespace, chartDir));
+
+    // build helm upgrade command
+    StringBuffer upgradeCmd = new StringBuffer("helm upgrade ")
+        .append(params.getReleaseName()).append(" ").append(chartDir)
+        .append(" --namespace ").append(params.getNamespace());
+
+    // add all the parameters
+    appendValues(params.getValues(), upgradeCmd);
+
+    upgradeCmd.append(" && helm list");
+    // run the command
+    return exec(upgradeCmd.toString());
   }
 
-  public String getRepoUrl() {
-    return repoUrl;
+  /**
+   * uninstall a helm release
+   * @param params the parameters to helm as values
+   * @return true on success, false otherwise
+   */
+  public static boolean uninstall(HelmParams params) {
+    // assertions for required parameters
+    assertThat(params.getNamespace())
+        .as("make sure namespace is not empty or null")
+        .isNotNull()
+        .isNotEmpty();
+
+    assertThat(params.getReleaseName())
+        .as("make sure releaseName is not empty or null")
+        .isNotNull()
+        .isNotEmpty();
+
+    return exec("helm uninstall " + params.getReleaseName()
+        + " -n " + params.getNamespace());
   }
 
-  public HashMap<String, Object> getValues() {
-    return values;
+  /**
+   * Add a chart repository
+   * @param chartName the name of the chart
+   * @param repoUrl reposiroty url
+   * @return true on success, false otherwise
+   */
+  public static boolean addRepo(String chartName, String repoUrl) {
+    String addRepoCmd = "helm add repo " + chartName + " " + repoUrl;
+    return exec(addRepoCmd);
   }
 
-  public boolean install() {
-    StringBuffer installCmd = new StringBuffer("helm install ")
-                              .append(releaseName).append(" ").append(chartName);
+  /**
+   * append the values to the given string buffer
+   * @param values hash map with key, value pairs
+   * @param command the command to append to
+   */
+  private static void appendValues(HashMap<String, Object> values, StringBuffer command) {
     values.forEach((key, value) ->
-                    installCmd.append(" --set ")
-                              .append(key)
-                              .append("=")
-                              .append(value.toString().replaceAll("\\[","{").replaceAll("\\]","}")));
-    logger.info("Running helm install command " + installCmd);
-    return true;
-  }
-
-  public boolean upgrade() {
-    return true;
-  }
-
-  public boolean delete() {
-    return true;
+        command.append(" --set \"")
+            .append(key)
+            .append("=")
+            .append(value.toString().replaceAll("\\[", "{").replaceAll("\\]", "}").replace(" ",""))
+            .append("\""));
   }
 
-  public boolean addRepo() {
-    String addRepoCmd = "helm add repo " + chartName + " " + repoUrl;
-    // execute addRepoCmd
+  /**
+   * Executes the given command
+   * @param command the command to run
```

Comment:
nitpick: if this method "executes" a command, then it is probably nicer to say "the command to execute"